<launch>
  <param name="/use_sim_time" value="false"/>
  <node pkg="trajectory_follower" type="mpc_node" name="mpc_node" output="screen" respawn="true">
	<rosparam  file="$(find trajectory_follower)/cfg/TrajectoryFollowerConfigMKz.yaml" command="load"/>
  </node>
</launch>
